cccore/Program.cs

```diff
@@ -537,6 +538,24 @@ public static List<string> Run(HttpClient httpClient, string creditcoinUrl, stri
                             Debug.Assert(msg != null || msg == null && link != null);
                             if (msg == null)
                                 return null;
+                            if (command[0].Equals("RegisterAddress", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I don't understand this if condition. This source file looks like an executable program, not a library so if this a shared entry point for other programs ? How is this consumed ?

Reply:
Currently, the method is only called through the ccclient executable. But earlier in the method, the first string in the command array is set to the `action` variable, and the array is shifted to the left.
The method probably needs to be refactored either here or in a later task. It's a large method (over 500 lines) and currently has too many responsibilities.